### PR TITLE
Design pages

### PR DESCRIPTION
**Pull request by mim24sd**
In this pull request we want to :
1. Complete Page Components.

### CODE REVIEW

File: index.css [hunk 1]
```diff
@@ -1,19 +1,6 @@
...
-p,
-input {
-  font-family: var(--primaryFont);
-  color: var(--primaryFontColor);
-}
-
-a {
-  cursor: pointer;
-}
+@import "./src/pages/Home/styles.css";
```
Comment (blacksrc) [inline index.css]: Shouldn't you import each css file in its own component?

File: src/components/MusicList/index.jsx [hunk 2]
```diff
@@ -1,5 +1,5 @@
 const MusicList = () => {
-  return <div></div>;
+  return <p>I'm a musiclist</p>;
```
Comment (blacksrc) [inline src/components/MusicList/index.jsx]: Not a most, but you could just write `Musicalist`. Would make more sense in a real project.

File: src/components/PlayLists/index.jsx [hunk 3]
```diff
@@ -1,5 +1,5 @@
 const PlayLists = () => {
-  return <div></div>;
+  return <p>I'm a playlist</p>;
```
Comment (blacksrc) [inline src/components/PlayLists/index.jsx]: Same:
https://github.com/mim24sd/music-app/pull/3/files#r1047538311

File: src/components/Sidebar/index.jsx [hunk 4]
```diff
@@ -1,5 +1,5 @@
 const Sidebar = () => {
-  return <div></div>;
+  return <p>I'm a sidebar</p>;
```
Comment (blacksrc) [inline src/components/Sidebar/index.jsx]: Same:
https://github.com/mim24sd/music-app/pull/3/files#r1047538311

Comment (blacksrc) [reply src/components/Sidebar/index.jsx:2]: ➕ Should be identical to design? Please add other items as well.
➕ Semantic html maybe?

File: src/components/Header/index.jsx [hunk 5]
```diff
@@ -2,7 +2,7 @@ import "./styles.css";
 
 const Header = ({ title }) => {
   return (
-    <header className="container">
+    <header>
       <h1 className="title">{title}</h1>
       <hr className="horizontal-line" />
```
Comment (takrami) [inline src/components/Header/index.jsx]: you can use border bottom instead.

File: src/pages/Albums/index.jsx [hunk 6]
```diff
@@ -1,7 +1,17 @@
 import Header from "../../components/Header";
+import MusicList from "../../components/MusicList";
+import Sidebar from "../../components/Sidebar";
 
 const Albums = () => {
-  return <Header title="Albums" />;
+  return (
+    <div className="main-container">
+      <Sidebar />
+      <div>
```
Comment (takrami) [inline src/pages/Albums/index.jsx]: avoid using div. use a semantic element instead.

File: src/pages/Home/index.jsx [hunk 7]
```diff
@@ -1,7 +1,17 @@
 import Header from "../../components/Header";
+import MusicList from "../../components/MusicList";
+import Sidebar from "../../components/Sidebar";
 
 const Home = () => {
-  return <Header title="New albums and singles" />;
+  return (
+    <div className="main-container">
+      <Sidebar />
+      <div>
```
Comment (takrami) [inline src/pages/Home/index.jsx]: avoid using div. use a semantic element instead.

File: src/pages/PlayLists/index.jsx [hunk 8]
```diff
@@ -1,7 +1,17 @@
 import Header from "../../components/Header";
+import PlayListsComponent from "../../components/PlayLists";
+import Sidebar from "../../components/Sidebar";
 
 const PlayLists = () => {
-  return <Header title="New Albums and Songs" />;
+  return (
+    <div className="main-container">
+      <Sidebar />
+      <div>
```
Comment (takrami) [inline src/pages/PlayLists/index.jsx]: avoid using div. use a semantic element instead.

File: src/pages/Singles/index.jsx [hunk 9]
```diff
@@ -1,7 +1,17 @@
 import Header from "../../components/Header";
+import MusicList from "../../components/MusicList";
+import Sidebar from "../../components/Sidebar";
 
 const Singles = () => {
-  return <Header title="Singles" />;
+  return (
+    <div className="main-container">
+      <Sidebar />
+      <div>
```
Comment (takrami) [inline src/pages/Singles/index.jsx]: avoid using div. use a semantic element instead.

File: src/components/MusicList/index.jsx (hunk 2 above)
Comment (mim24sd) [reply src/components/MusicList/index.jsx]: Thank that was a reaaaally good point

File: index.css (hunk 1 above)
Comment (mim24sd) [reply index.css]: You made me learn alot by this comment .
Thank you very muchhhhhhhhhhhh

File: src/components/Header/index.jsx (hunk 5 above)
Comment (mim24sd) [reply src/components/Header/index.jsx]: ok thanks

File: src/pages/Home/index.jsx (hunk 7 above)
Comment (mim24sd) [reply src/pages/Home/index.jsx:9]: thanksssssss

File: src/pages/Albums/index.jsx (hunk 6 above)
Comment (mim24sd) [reply src/pages/Albums/index.jsx]: thanksssssss

File: package.json [hunk 10]
```diff
@@ -19,11 +19,11 @@
     "eslint-plugin-jsx-a11y": "^6.5.1",
     "eslint-plugin-react": "^7.28.0",
     "eslint-plugin-react-hooks": "^4.3.0",
-    "parcel": "^2.2.1",
     "prettier": "^2.5.1",
     "process": "^0.11.10"
   },
   "dependencies": {
+    "parcel": "^2.8.2",
```
Comment (takrami) [inline package.json]: why do we need to have parcel in dependency?

Comment (mim24sd) [reply package.json]: sorry that was my mistake

**Pull request merged**